src/ray/raylet/scheduling/cluster_resource_scheduler.h

```diff
@@ -201,6 +201,13 @@ class ClusterResourceScheduler {
   /// Get number of nodes in the cluster.
   int64_t NumNodes();
 
+  /// Add an (unused) local resource.
```

Comment:
This description doesnt' sound clear to me. What does it mean by unused? Can you make it a little bit more clear?